common/data/dataHelpers.ts

```diff
@@ -0,0 +1,12 @@
+﻿/* eslint-disable */
+export const StrapiBaseUrl = "https://map-api.ekaterinburg.io/api";
+
+export async function getDataJsonByUrl(url: string){
+    return await fetch(url)
+        .then(x => x.json());
+}
```

Comment:
Let's remove this function and use fetch everywhere

Reply:
but i'll write it everywhere

Reply:
ok, I saw

Let's call it `fetchAPI()`

Reply:
```js
async function fetchAPI(url) {
   const response = await fetch(url);
   const data = await response.json();
}
```